source/common/crypto/utility.h

```diff
@@ -27,48 +27,46 @@ struct VerificationOutput {
 
 using PublicKeyPtr = bssl::UniquePtr<EVP_PKEY>;
```

Comment:
Could we get rid of this (and `openssl/evp.h`) from the interface? We shouldn't use any implementation-specific types here.

Reply:
If we get rid of evp.h then the verifySignature signature will have to change. Are you ok with that?

Reply:
Yes I'm in favor of that :+1:

Reply:
Yes, that's fine.